spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 # Coverage must be enabled before the application is loaded.
 if ENV['COVERAGE']
   require 'simplecov'
+  require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.